.buildkite/pipeline.build.yml

```diff
@@ -184,7 +184,7 @@
     - ./ci/env/env_info.sh
     - pip install ray[client]
     - bazel test --config=ci $(./ci/run/bazel_export_options)
-      --test_tag_filters=client_tests,small_size_python_tests
+      --test_tag_filters=client_tests,small_size_python_tests,-ci_blocking
```

Comment:
can we make blocking to be the default? and add a "civ1" tag for nonblocking?

Reply:
totally